docusaurus/docs/React/custom-code-examples/livestream-setup.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: livestream-setup
+sidebar_position: 22
```

Comment:
Should be sidebar_position: 21

Reply:
I think 22 is good, [because the previous article was 21](https://github.com/GetStream/stream-chat-react/blob/680bdc4f9e662fd64306b568fb16a92e09a259ba/docusaurus/docs/React/custom-code-examples/channel-users-list.mdx#L3)?

Reply:
You're right, I was comparing to the published version, but the newest user presence file isn't there yet.